strategy-agent/tests/unit/test_storage.py

```
@@ -0,0 +1,240 @@
+
+            mock_bucket.blob.return_value = mock_blob
+            mock_client.bucket.return_value = mock_bucket
+            mock_client_class.return_value = mock_client
+
+            service = RealStorageService()
+
+            # Test PNG
+            content = b"png data"
+            content_type = "image/png"
+
+            url = await service.upload_reference_image("event123", content, content_type)
```

Ruff (0.14.3)

Local variable `url` is assigned to but never used

Remove assignment to unused variable `url`

(F841)